### ISSUE TITLE
Jetpack Scan: update the way the public key is shown

### ISSUE THREAD

**Issue by v18** [Automattic/wp-calypso#168]
The string for the public key is hard to copy and paste:

![](https://cloud.githubusercontent.com/assets/115071/8383469/6d9a5998-1bef-11e5-89dd-81afc9a9b704.png)

Notes on what could be improved:
- I would like to see the whole string and also more instructions on how to do it
- A link to documentation would help
- instead of a dialog modal I think we should use an expanding box. Modals don't work if the content is to big

**Comment by designsimply** [comment]
Tested and confirmed with the following steps:

1. Go to https://wpcalypso.wordpress.com/settings and select a Jetpack-connected site
1. Click Security in the header bar
1. Click Activate in the Jetpack Scan box if the settings aren't already visible
1. Click the "Get your public key string." link (bottom of the box)
1. Try to copy and paste the string

<strong>Result:</strong> when clicking on the form field containing the string, the entire string is highlighted so I just had to press `cmd + c` to copy it—I found this easy to do.

Recommend closing. May I have a 2nd opinion?
